notebooks/dataAccess/siphon-RemoteAccess.ipynb

```diff
@@ -0,0 +1,408 @@
+    "catUrl = \"https://thredds-test.unidata.ucar.edu/thredds/catalog/grib/nexrad/composite/unidata/NEXRAD_Unidata_Reflectivity-20201207/Level3_Composite_N0R_20201207_0000.grib2/catalog.xml\";\n",
+    "datasetName = \"Level3_Composite_N0R_20201207_0000.grib2\";"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If you have another TDS dataset in mind, you can replace the catlog URL and dataset name above to point to that dataset instead."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next, we access the catalog using the catalog URL:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "catalog = TDSCatalog(catUrl)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "And then select our dataset using the dataset name:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ds = catalog.datasets[datasetName]\n",
+    "ds.name"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can now view the access protocols available for our dataset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "list(ds.access_urls)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "<a href=\"#top\">Top</a>\n",
+    "\n",
+    "---"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2. Access the dataset using `remote_access`\n",
+    "\n",
+    "Now that we have our dataset and know its access protocols, we can access the remote dataset."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Access via CDM Remote\n",
+    "If the name of the service is not provided, `remote_access` defaults to using the `CdmRemote` service."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dataset = ds.remote_access()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The call to `ds.remote_access` opens the remote dataset and returns a netCDF4-like dataset object, which provides access to the metadata."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# list attributes\n",
+    "list(dataset.ncattrs())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# list variables\n",
+    "list(dataset.variables)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Access via OPENDAP\n",
+    "We can also use `remote_access` to open the dataset via OPENDAP."
```

Comment:
Does this also return data in the netcdf format?  Might just be good to clarify since you provided that information for the default method of accessing data.
Are there any noticeable difference between using the default method and specifying OPENDAP?  Are there pros and cons that would be useful for the user to know about?